pom.xml

```diff
@@ -47,6 +45,20 @@
               <transformationSet>
                 <dir>.</dir>
                 <includes>
+                  <include>indexing.rdf</include>
```

Comment:
I haven't added it yet. It will come with the next commit -- it needed a bit of modification to work with rdf:Property and rdfs:Class